packages/grafana-runtime/src/config.ts

```diff
@@ -182,6 +182,7 @@ export class GrafanaBootConfig implements GrafanaConfig {
   localFileSystemAvailable: boolean | undefined;
   cloudMigrationIsTarget: boolean | undefined;
   cloudMigrationFeedbackURL = '';
+  cloudMigrationPollIntervalMs = 2000;
```

Comment:
am i right in thinking it only takes this value if the backend value isn't set at all? 🤔

Reply:
Yeah... the "defaults in code" (both this file and in the backend when parsing config) is kinda weird as they're never used in practice because we (should) have the defaults.ini file.